src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String text, String fileName) {

```

Comment:
As per the checklist, avoid hardcoding the file path in your solution. It's good that you're taking the file path as a parameter, which allows for flexibility and adherence to the checklist's best practices.